### ISSUE TITLE
Problem with covers in artist list

### ISSUE THREAD

**Issue by Hurqalia** [Taiko2k/TauonMusicBox#297]
Sometime, the covert cannot be created and displayed. One of these cases occurs when the artist name contains a slash, like AC/DC for example. Is it possible to ignore (replace by nothing) slash in load, get_data etc... each time the artist name is used in a filename ?  Thanks for all, tauon rocks ;-)

**Comment by Taiko2k** [comment]
Okay, I've committed some changes. 

I tested with AC/DC and that seems to be working now. Also artists with "&" should be more likely to work now. There might still be other cases not covered, you might have you tell me for any individual case not working.

Though, if an artist image isn't appearing, its most likely because there isn't one. Obtaining a Discogs token if you haven't already should help. 

BTW, when you upgrade next, you'll want to go `MENU` > `Database...` > `Reset Image Cache` to trigger a retry of those failed images. (Don't worry, it wont wipe existing downloaded artist images)
